### DISCUSSION TITLE
Linking my work account with my personal account

### DISCUSSION THREAD

**Discussion by rohankokkulabito** [community/community#133322] [Other Feature Feedback, Questions, & Ideas]
Hi github community members, 
So I am looking for a way to connect my github work account with my personal github account.
I have been getting the error "email is already in use"
Any ways to fix this?

**Comment by AtlasKeeper** [comment]
The error "email is already in use" occurs when you try to use an email address that is already associated with another GitHub account. GitHub requires unique email addresses for each account to avoid confusion.

**Reply by rohankokkulabito** [reply]
So there's no way to do it? 
Will it work by deactivating the account and trying again?
